transit_odp/common/xmlelements/elements.py

```diff
@@ -190,3 +190,8 @@ def parent(self):
     def line_number(self):
         """Returns the line number of the element."""
         return self._element.sourceline
+
+    @property
+    def tag_localname(self):
```

Comment:
Can we reuse the existing localname function?

Reply:
done